drivers/gles2/shaders/scene.glsl

```diff
@@ -1547,157 +1545,157 @@ FRAGMENT_SHADER_CODE
 #endif // !USE_SHADOW_TO_OPACITY
 
 #ifdef BASE_PASS
-	{
```

Comment:
Didn't you add this extra scope a few commits ago in another PR? :)

Reply:
Yes, but it made the code messier, and it is unnecessary now that I removed the environment scaling from the vertex_lighting code path